packages/cfd/src/Components/__tests__/cfd-poa.spec.js

```diff
@@ -0,0 +1,296 @@
+    WS: {
+        authorized: {
+            getAccountStatus: jest.fn().mockResolvedValue({ get_account_status: 1 }),
+        },
+        wait: (...payload) => {
+            return Promise.resolve([...payload]);
+        },
+    },
+}));
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(),
+}));
+
+describe('<CFDPOA />', () => {
+    const address_line_1 = 'First line of address*';
+    const address_line_2 = 'Second line of address';
+    const address_postcode = 'Postal/ZIP code';
+    const address_state = 'State/Province';
+    const address_town = 'Town/City*';
+
+    let props;
+
+    beforeEach(() => {
+        isMobile.mockReturnValue(false);
+
+        props = {
+            onSave: jest.fn(),
+            onCancel: jest.fn(),
+            index: 1,
+            onSubmit: jest.fn(),
+            refreshNotifications: jest.fn(),
+            form_error: '',
+            get_settings: {
+                account_opening_reason: '',
+                address_city: 'Woodlands',
+                address_line_1: "69 Test Street, .'",
+                address_line_2: ".'",
+                address_postcode: '666',
+                address_state: '',
+                allow_copiers: 0,
+                citizen: '',
+                client_tnc_status: 'Version 4.2.0 2020-08-07',
+                country: 'Singapore',
+                country_code: 'sg',
+                date_of_birth: 984960000,
+                email: 'mock@gmail.com',
+                email_consent: 1,
+                feature_flag: {
+                    wallet: 0,
+                },
+                first_name: 'thisyahlen',
+                has_secret_answer: 1,
+                immutable_fields: ['residence'],
+                is_authenticated_payment_agent: 0,
+                last_name: 'lol',
+                non_pep_declaration: 1,
+                phone: '+790875616',
+                place_of_birth: null,
+                preferred_language: 'EN',
+                request_professional_status: 0,
+                residence: 'Singapore',
+                salutation: '',
+                tax_identification_number: null,
+                tax_residence: null,
+                user_hash: '823341c18bfccb391b6bb5d77ab7e6a83991f82669c1ba4e5b01dbd2fd71c7fe',
+            },
+            height: 'auto',
+            is_loading: false,
+            states_list: {
+                text: 'Central Singapore',
+                value: '01',
+            },
+            storeProofOfAddress: jest.fn(),
+            value: {
+                address_city: '',
+                address_line_1: '',
+                address_line_2: '',
+                address_postcode: '',
+                address_state: 'Default test state',
+                poa_status: 'verified',
+            },
+        };
+    });
+
+    it('should render the POA component', async () => {
+        render(<CFDPOA {...props} />);
```

Comment:
Isn't this test case covered in the one below?
Why do you have to repeat it, yet no props have changed?

Reply:
Alrights, removed the above one